scripts/build-info.js

```diff
@@ -2,10 +2,9 @@
 
 const fs = require('fs')
 const { dirname } = require('path')
-const mkdirp = require('mkdirp')
 
 function writeFile(path, contents, callback) {
-  mkdirp(dirname(path), (err) => {
+  fs.mkdir(dirname(path), { recursive: true }, (err) => {
```

Comment:
can use `fs.mkdir` directly since node 10.  `mkdirp` had been upgraded to v1 which was a breaking change, so chance to remove instead.